source/beamline/SGM/SGMBeamline.cpp

```diff
@@ -202,6 +207,10 @@ void SGMBeamline::setupBeamlineComponents()
 	hexapod_ = new SGMHexapod(this);
 	// Rotate the hexapod system to the plane of the sample plate.
 	hexapod_->rotateSystem(45, 0, 105);
+	// Control set for scan configurations.  Probably need a better solution.
+	hexapodControlSet_ = new AMControlSet(this);
+	hexapodControlSet_->addControl(hexapod_->xAxisPrimeControl());
+	hexapodControlSet_->addControl(hexapod_->zAxisPrimeControl());
```

Comment:
X` Y` 
I spoke with Tom regarding whether he wanted to to switch the nomenclature from the hexapod standard to the CLS standard, and he requested I kept it how it is. 
As such X moves horizontally across the sample plate, Y moves vertically up/down the sample plate and Z moves the sample plate closer to/further away from the microscope.